ui/src/logs/components/TimeRangeDropdown.tsx

```diff
@@ -132,6 +128,21 @@ class TimeRangeDropdown extends Component<Props, State> {
     )
   }
 
+  private get dropdownClassName(): string {
+    const {
+      isOpen,
+      customTimeRange: {lower, upper},
+    } = this.state
+
+    const absoluteTimeRange = !!lower || !!upper
```

Comment:
Alex what do you think of using `!_.isEmpty(lower)` here and the same for upper?